src/App.js

```diff
@@ -191,10 +202,10 @@ class App extends Component {
     this.state = INITIAL_STATE;
     const url = new URL(window.location);
     const urlParams = new URLSearchParams(url.search);
-    const partyId = urlParams.get('party') || localStorage.getItem('party.id');
     const tokenCookiePair = document.cookie.split('; ').find(row => row.startsWith('DABL_LEDGER_ACCESS_TOKEN')) || '';
     const tokenCookieSecret = tokenCookiePair.slice(tokenCookiePair.indexOf('=') + 1);
     const token = tokenCookieSecret || urlParams.get('token') || localStorage.getItem('party.token');
+    const partyId = urlParams.get('party') || partyFromToken(token) || localStorage.getItem('party.id');
```

Comment:
Are you sure this is what you want, in the sense that it prefers the presence of the url param to the parsed token party?

I can't imagine a scenario where an id is passed via a get parameter, and it needs to be used in preference to a token field - the get parameter was always redundant, so why not remove ```urlParams.get('party')``` entirely?